### PR TITLE
Encryption FAQ improvements, adapt to verified 1:1 chats

### PR DESCRIPTION
**Pull request by Hocuri**
Adapt FAQ for https://github.com/deltachat/deltachat-core-rust/issues/4188

I want to push this forward already now because when you tap on "Learn More", then the FAQ opens. But for now, the relevant FAQ section isn't written yet. When we settled for a rough version, holger had the idea to put it into some hidden place on the website, so that it can be linked from the Android app, but won't be translated yet since we may still want to change things.

### CODE REVIEW

File: en/help.md [hunk 1]
```diff
@@ -267,113 +267,109 @@ it is recommended to leave any group chat before uninstalling Delta Chat.
...
+email provider and the email provider of your chat partner can read and modify your emails.
+Your email provider is what comes after the `@` in your email address
+(gmail.com, outlook.com, …)
 
-- Delta Chat apps (and other [Autocrypt](https://autocrypt.org)-compatible
-  e-mail apps) share the keys required for end-to-end-encryption automatically 
-  as the first messages are sent. 
-  After this, all subsequent messages are encrypted end-to-end automatically. 
-  If one of the chat partners uses a non-Autocrypt e-mail app, subsequent 
-  messages are not encrypted until an Autocrypt-compliant app is available again.
```
Comment (hpk42) [inline en/help.md:286]: this bit was lost, no?   I think the opportunistic character of DC's default end-to-end encryption provides another reason (apart from the hacking/MITM attack on the side of e-mail providers) to use verified chats.

File: en/help.md [hunk 2]
```diff
@@ -267,113 +267,109 @@ it is recommended to leave any group chat before uninstalling Delta Chat.
...
 
-### How can I check the encryption of messages?
+### "Messages may not be end-to-end encrypted anymore", what does this mean? {#verificationbroken}
 
-- A little **padlock** shown beside a message denotes whether the message is end-to-end-encrypted from from the given sender.
+At some point in the past, you probably [scanned a QR Code](#howtoe2ee), which ensured that all your messages
+were correctly end-to-end encrypted.
 
-- If there is **no padlock**, the message is usually transported unencrypted e.g. because you or the sender have turned off end-to-end-encryption, or the sender uses an app without support for end-to-end-encryption.
+Now, somehow your chat partner's encryption key changed; reasons for this are:
```
Comment (Hocuri) [inline en/help.md:338]:
```suggestion
Now, somehow your chat partner's encryption key changed; possible reasons for this are:
```

File: en/help.md [hunk 3]
```diff
@@ -267,113 +267,109 @@ it is recommended to leave any group chat before uninstalling Delta Chat.
...
+- They reinstalled Delta Chat
+- They are using a new phone, and didn't properly [transfer their account](#multiclient)
+- They didn't use Delta Chat to send the message, but sent a classical email.
```
Comment (Hocuri) [inline en/help.md:341]: Maybe also mention the possibility of a MitM attack here?

**Comment by hpk42** [comment]
closing this in favor of the merged https://github.com/deltachat/deltachat-pages/pull/727
could you do a new PR with anything you feel is missing from this PR here?

**Pull request closed without merging**